sunpy/data/sample.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+"""SunPy sample data files"""
+
+import sys as _sys
```

Comment:
the difference is that the `__all__` will actually change what is imported, your way just asks things to ignore it.